docs/source/advanced/plugin-configuration.mdx

```diff
@@ -205,6 +205,35 @@ by default).
 
 With the [Android Studio plugin](../testing/android-studio-plugin), you can also go to <kbd>Tools</kbd> | <kbd>Apollo</kbd> | <kbd>Download schema</kbd> which acts as a shortcut to these tasks.
 
+## Generated methods
+By default all Kotlin models, operations, fragments, and input objects are generated as data classes. This means that the compiler will
+auto-generate `toString`, `equals` `hashCode`, `copy` and `componentN` for most generated classes. If you don't think you need all of those
+auto-generated methods, and/or you are worried about the size of the generated code, you can configure the Apollo compiler to generate none
+or a subset of the data class methods. To do this, set `generateMethods` to a list of the methods you need. The available methods are:
```

Comment:
Reminder to myself to quantify this, maybe using Confetti to get a rough idea of the expected improvement.